### ISSUE TITLE
hyper 0.11?

### ISSUE THREAD

**Issue by mssun** [derniercri/snatch#76]
Compared to 0.9, I guess hyper dropped openssl dependency and made it a pure rust library.

I wonder when snatch will move on to hyper 0.11 and give an option to decide whether to use openssl. OpenSSL can potentially introduce security issue.

This will give users an option to use pure Rust binary.

**Comment by k0pernicus** [comment]
Hello @mssun,
Actually, I think Snatch is no longer maintained.
@Jean-Serge, can you confirm please?

Concerning your issue, I am agree with you.
I forked Snatch as Zou (https://github.com/k0pernicus/zou) - maybe you can create an issue in the project if you want, and/or create a PR for this issue.

Thanks a lot for reporting issues!

**Comment by mssun** [comment]
Ok, really?

Actually, I am looking for a pure-rust wget/curl tool.

**Comment by k0pernicus** [comment]
Yep, that's the plan for `zou` - to be the next `wget`: most simple, most secure, most efficient.

**Comment by mssun** [comment]
@k0pernicus Issue copied to you fork. Thanks.

**Comment by k0pernicus** [comment]
@mssun Got it - I will work on that soon ;-)

**Comment by mssun** [comment]
Awesome. Thanks.
